tfe/resource_tfe_workspace.go

```diff
@@ -211,6 +211,11 @@ func resourceTFEWorkspace() *schema.Resource {
 							Type:     schema.TypeString,
 							Required: true,
 						},
+
+						"tags_regex": {
+							Type:     schema.TypeString,
+							Optional: true,
```

Comment:
We'll need to add a `ConflictsWith: []string{"trigger_prefixes", "trigger_prefixes", "file_triggers_enabled"}` to keep this in sync with the API